ee/clickhouse/models/cohort.py

```diff
@@ -52,3 +70,15 @@ def get_person_ids_by_cohort_id(team: Team, cohort_id: int):
     results = sync_execute(GET_PERSON_IDS_BY_FILTER.format(distinct_query=filter_query, query=""), filter_params)
 
     return [str(row[0]) for row in results]
+
+
+def insert_static_cohort(person_uuids: List[Optional[uuid.UUID]], cohort_id: int, team: Team):
+    for person_uuid in person_uuids:
+        pb_event = person_static_cohort_pb2.PersonStaticCohort()
+        pb_event.id = str(uuid.uuid4())
+        pb_event.person_id = str(person_uuid)
+        pb_event.cohort_id = cohort_id
+        pb_event.team_id = team.pk
+
+        p = ClickhouseProducer()
+        p.produce_proto(sql=INSERT_PERSON_STATIC_COHORT, topic=KAFKA_PERSON_STATIC_COHORT, data=pb_event)
```

Comment:
This is perfect for a proto encoded table, but the table you have defined is json encoded.

Check out producing for persons to see what producing to a json table looks like
https://github.com/PostHog/posthog/blob/master/ee/clickhouse/models/person.py#L74